File: TGAPEngine/src/Keyframe.cpp
#include "Keyframe.h"

namespace engine{
	Keyframe::Keyframe(unsigned int frame, glm::vec3 location, glm::quat quaternion)
	{
		this->frame = frame;
		this->location = location;
		this->quaternion = quaternion;
	}


	Keyframe::~Keyframe()
	{
	}
}

File: CS/TGAPE_CS/Engine/Game/Keyframe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Numerics;
using System.Text;
using System.Threading.Tasks;

namespace TGAPE_CS.Engine.Game
{
    public struct Keyframe
    {
        public uint frame;
        public Quaternion rotation;
        public Vector3 location;
        public Keyframe(uint frame, Vector3 location, Quaternion rotation)
        {
            this.frame = frame;
            this.location = location;
            this.rotation = rotation;
        }
    }
}
